wenet/utils/common.py

```diff
@@ -83,6 +84,32 @@ def add_sos_eos(ys_pad: torch.Tensor, sos: int, eos: int,
     ys_out = [torch.cat([y, _eos], dim=0) for y in ys]
     return pad_list(ys_in, eos), pad_list(ys_out, ignore_id)
 
+def reverse_pad_list(ys_pad: torch.Tensor, ys_lens: torch.Tensor,
+                     pad_value: float = -1.0) -> torch.Tensor:
+    """Reverse padding for the list of tensors.
+
+    Args:
+        ys_pad (tensor): The padded tensor (B, Tokenmax).
+        ys_lens (tensor): The lens of token seqs (B)
+        pad_value (int): Value for padding.
+
+    Returns:
+        Tensor: Padded tensor (B, Tokenmax).
+
+    Examples:
+        >>> x
+        tensor([[1, 2, 3, 4], [5, 6, 7, 0], [8, 9, 0, 0]])
```

Comment:
use the same style  as the following tensor
tensor([[1, 2, 3, 4], 
              [5, 6, 7, 0], 
              [8, 9, 0, 0]])